Backpack-SwiftUI/Tests/Icons/BPKIconViewTests.swift

```diff
@@ -18,6 +18,7 @@
 
 import XCTest
 import SwiftUI
+import SnapshotTesting
```

Comment:
I think you don't need the import here, since the assertSnapshot that receives a SwiftUI view is a helper method on our module